pygfx/resources/_texturemap.py

```diff
@@ -117,3 +133,90 @@ def wrap_t(self):
     @wrap_t.setter
     def wrap_t(self, value):
         self._store.wrap_t = value
+
+    @property
+    def offset(self):
+        """The offset of the texture map.
```

Comment:
Can we add a note in the docstring of TextureMap or in the docstring of these new props, that they currently only apply to Mesh objects?

Reply:
Theoretically, this should be a property of TextureMap and should apply to all 2D textures. However, at the moment, it only affects Mesh. We can document this limitation accordingly.